### PR TITLE
Modified API Docs documentation

### PR DESCRIPTION
**Pull request by Irma12**
Modified API Docs documentation

### CODE REVIEW

File: content/backstage/plugins/api-docs.md [hunk 1]
```diff
@@ -24,28 +24,103 @@ gettingStarted:
...
+
+        export const entityPage = (
+          <EntitySwitch>
+            // ...
+            <EntitySwitch.Case if={isKind('api')} children={apiPage} />
+            // ...
+          </EntitySwitch>
         );
+
+  - intro: Custom API Renderings
```
Comment (dtuite) [inline content/backstage/plugins/api-docs.md:84]: Could you expand on what this means please? I can't understand what the result of this step will be.

Comment (Irma12) [reply content/backstage/plugins/api-docs.md:84]: I followed mostly this https://www.npmjs.com/package/@backstage/plugin-api-docs but I will expand with explanation

File: content/backstage/plugins/api-docs.md [hunk 2]
```diff
@@ -21,31 +21,72 @@ coverImageAlt: 'A screenshot of the API Docs. It is showing a available endpoint
...
+  - language: bash
+    code: |
+      The plugin is already added when using `npx @backstage/create-app` 
+      so you can skip these steps. However, if you are not using create-app
+      you can follow the steps below.
   - intro: Install the plugin into Backstage.
     language: bash
     code: 'yarn add @backstage/plugin-api-docs'
-  - intro: Add plugin to the list of plugins.
+  - intro: Add the ApiExplorerPage extension to the app:.
```
Comment (dtuite) [inline content/backstage/plugins/api-docs.md:32]: There seems to be a typo full stop at the end of this line.

```suggestion
  - intro: Add the ApiExplorerPage extension to the app:
```

**Review by dtuite — COMMENTED** [review]
Just one tiny typo. Looks good other than that.

**Pull request merged**